keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ResourceController.kt

```diff
@@ -51,46 +55,53 @@ class ResourceController(
 ) {
 
   private val log by lazy { LoggerFactory.getLogger(javaClass) }
+  private val scope: CoroutineScope
+    get() = CoroutineScope(IO)
```

Comment:
All the endpoints in this controller _are_ doing database IO so this seems like the right dispatcher.